_includes/team.html

```diff
@@ -24,6 +24,11 @@ <h6 class="card-subtitle mb-2 text-muted" itemprop="jobTitle">{{ include.role |
         <i class="fas fa-link" aria-hidden="true" title="Website"></i>
       </a>
       {% endif %}
+      {% if include.keys %}
+      <a href="{{ include.keys }}" aria-label="Keys" class="card-link" itemprop="url">
```

Comment:
```suggestion
      <a href="{{ include.keys }}" aria-label="Keys" class="card-link" itemprop="sameAs">
```

https://schema.org/Person